### DISCUSSION TITLE
🇩🇪 Das Drachental

### DISCUSSION THREAD

**Discussion by kironia** [eltikia/eltikia.github.io#6] [Q&A] [answered]
Was bedeute die Videobeschreibungen?

**Comment by Stagyrite** [comment, marked as answer]
Das ist ein Beispiel. Das Spiel kommuniziert nur mit Text. Deshalb ein Satz oder ein Wort kann ein Beispiel sein. Ich brauche kein Bild.

> Wenn „trowrebuaz“ gesagt wird, ein Schwert ist frei.

Das ist passiert.

> Das ist ganz normal in dieses Computerspiel.

Richtig.

> „Trowrebuaz“ ist ein Zauberwort.

„Trowrebuaz“ ist „Zauberwort“ rückwärts buchstabiert.
